localstack/services/opensearch/provider.py

```diff
@@ -157,6 +157,9 @@ def create_cluster(
         preferred_port=preferred_port,
     )
 
+    for k, v in store.opensearch_domains.items():
+        v["AdvancedSecurityOptions"]
+
```

Comment:
Since this isn't really doing anything, this line might be a leftover from testing?

Reply:
Yep, sorry for the unnecessary ping!